HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -17,6 +17,10 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+# MultipleScatteringParametrisationMakerESProducer
+def customiseFor35269(process):
+    process.load("RecoTracker.TkMSParametrization.multipleScatteringParametrisationMakerESProducer_cfi")
```

Comment:
I haven't seen any specific imports in the HLT configuration; in the configBuilder-based configurations the extra ES configs come via Configuration/StandardSequences
In this case CalibTracker/Configuration/python/Tracker_DependentRecords_forGlobalTag_nofakes_cff.py (included by Configuration/StandardSequences/python/AdditionalConditions_cff.py, included by the standard Configuration/StandardSequences/python/FrontierConditions_GlobalTag_cff.py) seems to be an appropriate place to insert

Reply:
The new ESProducer needs to eventually go in the HLT menu as well.

Reply:
Or does it actually?

Reply:
The ESProducer is needed in the HLT menu. Trying to run addOnTests without the customise function results in 
```
----- Begin Fatal Exception 20-Sep-2021 21:15:36 CEST-----------------------
An exception of category 'NoProxyException' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 3
   [1] Running path 'HLT_IsoTrackHE_v4'
   [2] Calling method for module HitPairEDProducer/'hltPixelTracksHitDoublets'
Exception Message:
No data of type "MultipleScatteringParametrisationMaker" with label "" in record "NavigationSchoolRecord"
 Please add an ESSource or ESProducer to your job which can deliver this data.
----- End Fatal Exception -------------------------------------------------
```
error in e.g. `hlt_mc_GRun` test.

Reply:
I see, it's the addOnTests specifics. Got it.

Reply:
Yes, we'd add it to the HLT menu via CobfDB eventually. The customisation is just for this PR.